classes/webservice/WebserviceRequest.php

```diff
@@ -308,11 +308,12 @@ public static function getResources()
             'messages' => array('description' => 'The Messages', 'class' => 'Message'),
             'order_carriers' => array('description' => 'The Order carriers', 'class' => 'OrderCarrier'),
             'order_details' => array('description' => 'Details of an order', 'class' => 'OrderDetail'),
-            'order_histories' => array('description' => 'The Order histories', 'class' => 'OrderHistory'),
-            'order_invoices' => array('description' => 'The Order invoices', 'class' => 'OrderInvoice'),
-            'orders' => array('description' => 'The Customers orders', 'class' => 'Order'),
-            'order_payments' => array('description' => 'The Order payments', 'class' => 'OrderPayment'),
-            'order_states' => array('description' => 'The Order statuses', 'class' => 'OrderState'),
+            'order_histories' => array('description' => 'The Order histories','class' => 'OrderHistory'),
```

Comment:
Please don't change all lines. Make sure to edit only the needed one. Here, you break the Php-cs-fixer test :)